Markup.php

```diff
@@ -54,21 +54,18 @@ public static function createElement($tag = '')
     /**
      *
      * Add element at an existing Markup
-     * @param Markup $tag
+     * @param Markup|string $tag
      * @return Markup instance
      */
     public function addElement($tag)
     {
-        $htmlTag = null;
```

Comment:
To be clean, you've to keep this variable declaration outside the "if/else" statement because it returned at the end of the method and this variable has no declaration of context otherwise.